quarkus/runtime/pom.xml

```diff
@@ -458,6 +458,17 @@
             <artifactId>infinispan-client-hotrod-jakarta</artifactId>
             <version>${infinispan.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-query-dsl</artifactId>
```

Comment:
This dependency was removed from this pom.xml, probably due to a mistake. Reverted.